packages/eas-cli/src/graphql/queries/AppleTeamQuery.ts

```diff
@@ -0,0 +1,35 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { AppleTeam } from '../generated';
+
+type AppleTeamQueryResult = Pick<AppleTeam, 'id' | 'appleTeamIdentifier' | 'appleTeamName'>;
+
+const AppleTeamQuery = {
+  async allForAccountAsync(accountName: string): Promise<AppleTeamQueryResult[]> {
```

Comment:
Maybe move this function to `packages/eas-cli/src/credentials/ios/api/graphql/queries/AppleTeamQuery.ts`